public_html/wp-content/plugins/pattern-directory/javascript/pattern-post-type.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * There's no need for webpack etc, since we can assume wp-admin users will have modern browsers.
+ *
+ * JSX is the only thing that'd be nice to have, but it's not worth the tooling costs for just a few fields.
```

Comment:
I can see the appeal of everything using the same tooling, but it still feels to me like this doesn't _need_ any tooling, and that the devex is better without me.

The existing tooling might might it easier to set this up, but it could still be a pain. The biggest cost IMO is _using_ the tooling, though. It's slow, it's breaks often and is painful to fix, etc.

This is already working, too, so I don't see any benefit from redoing it now. If we want to add more fields or components later, then I can see that making the plugin complex enough where JSX would be worth the costs.